classes/battle/BattleManager.php

```diff
@@ -880,8 +913,28 @@ protected function applyAttack(BattleAttack $attack, Fighter $user, Fighter $tar
                                 </p>"; }
                     }
 
+        if ($attack->recoil_percent > 0) {
+            if ($recoil_damage_resisted > 0) {
+                $text .= "<span>-" . $user->getName() . " takes <span class=\"battle_text_{$attack->jutsu->jutsu_type}\">" . round($recoil_damage, 0) . "</span> recoil damage- (resists " . "<span class=\"battle_text_{$attack->jutsu->jutsu_type}\">" . round($recoil_damage_resisted) . "</span>" . " recoil damage)" . '</span></br>';
```

Comment:
IMO especially if resist is a % now, we might be able to leave the resist part of this display off and just show the final recoil daamge

I think the absolute number was much more important when resist was not a % and it was hard to tell how much you would really resist